Menus/SettingsGUI.cpp

```diff
@@ -41,7 +41,14 @@ namespace RTE {
 		m_GameplaySettingsMenu = std::make_unique<SettingsGameplayGUI>(m_GUIControlManager.get());
 		m_MiscSettingsMenu = std::make_unique<SettingsMiscGUI>(m_GUIControlManager.get());
 
-		SetActiveSettingsMenuScreen(SettingsMenuScreen::VideoSettingsMenu, false);
+		if (createForPauseMenu) {
+			m_SettingsTabberBox->SetPositionAbs((rootBox->GetWidth() - m_SettingsTabberBox->GetWidth()) / 2, (rootBox->GetHeight() - m_SettingsTabberBox->GetHeight() - 30) / 2);
```

Comment:
Why did you reorder the settings gui for the ingame pause menu? I'm not saying it's a bad thing necessarily, I'm just curious - why reorder it for this but not the main menu?

Reply:
Idk, just felt like gameplay settings are the thing people are likely to want to mess with in-game, and it's nicer when it's the first tab and not the 4th.

Reply:
Sure, but might as well rearrange the main one too. I think most games start you off in gameplay settings, and that makes sense imo

Reply:
No thanks lel